src/Sql/Sql/ManagedInstance/Cmdlet/SetAzureSqlManagedInstance.cs

```diff
@@ -130,6 +130,25 @@ public class SetAzureSqlManagedInstance : ManagedInstanceCmdletBase
             HelpMessage = "Determines how much VCore to associate with instance")]
         public int? VCore { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether or not the public data endpoint is enabled.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "Whether or not the public data endpoint is enabled for the instance.")]
+        [ValidateNotNullOrEmpty]
+        [PSArgumentCompleter(Constants.PublicDataEndpointEnabledTrue, Constants.PublicDataEndpointEnabledFalse)]
+        public bool? PublicDataEndpointEnabled { get; set; }
```

Comment:
shouldn't this be `SwitchParameter` ?

Reply:
As @maddieclayton suggested in the design review I used SwitchParameter in "New" cmdlet. And he said that using boolean for "Set" cmdlet is correct syntax.